packages/react/types/jsx-namespace.d.ts

```diff
@@ -2,8 +2,12 @@ import 'react'
 import { Interpolation } from '@emotion/serialize'
 import { Theme } from './index'
 
-type WithConditionalCSSProp<P> = 'className' extends keyof P
-  ? (P extends { className?: string } ? P & { css?: Interpolation<Theme> } : P)
+// prettier-ignore
```

Comment:
Any reason why this is ignored here? Its in the source code so formatting shouldnt affect much

Reply:
Sorry, this isn't necessary anymore. Thanks!